protocol/lib/ante/nested_msg.go

```diff
@@ -2,73 +2,99 @@ package ante
 
 import (
 	"fmt"
+	"strings"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	authz "github.com/cosmos/cosmos-sdk/x/authz"
 	gov "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/dydxprotocol/v4-chain/protocol/lib/metrics"
 )
 
+const DYDX_MSG_PREFIX = "/dydxprotocol"
+
 // IsNestedMsg returns true if the given msg is a nested msg.
 func IsNestedMsg(msg sdk.Msg) bool {
 	switch msg.(type) {
```

Comment:
Question: do we need `MsgDelayMsg` here?

Reply:
good callout. I think so since it also wraps other messages. 

Let's add this in a separate PR since it's unrelated. I also have less context on delay messages so maybe one of the OTE eng can pick this up?